Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+e-mail address on a project or team membership on a project take around
+a day to
+propagate.
+
+## I use BOINC with one username but on a few machines - is it enough to have one wallet with the proper boinc\_project\_email address, or do I have to have a separate wallet on each machine?
+
+You only need one wallet for each BOINC account/CPID (Cross Project
+Identifier). You can use BAM (BOINC Account Manager) to combine more
+projects in one CPID. Here is a guide
+<http://grcnation.com/en/bam-boinc-account-manager/> . Your reward is
+based on the credit in your BOINC account/CPID. You can point as many
+machines and resources to a single account as you want and only need a
+single wallet to receive your reward. You do have to set "email=" in the
+config file to your BOINC/BAM email and must be running the wallet on a
+machine that has BOINC installed and is associated with all BOINC
+projects you would like to get credit for (they can be set to not
+retrieve new work) and each project must show at least some RAC on that
+machine (press "Update" in the BOINC software when a project which shows
+0 contribution is
+selected).
+
+## BOINC: For our accounts, we used to have to use our Gridcoin address as our usernames for those accounts. Is that the case still, and if so, should we change our address info over to the "new" client addresses? Like under BAM/BOINC Stats, our stats/info isn't searchable via e-mail address but is via username. Just wondering what we can/should use for going forward.
+
+Changing your BOINC name to a Gridcoin Address is not needed anymore.
+Now just set "email=" in the gridcoinresearch.config file, ensure you
+are in team Gridcoin in each project, that should do
+it.
+
+## Instead of discussing every single BOINC project to be added, is there any shared guidelines to be used for accepting or rejecting them?
+
+No, but there may be a defined process one day. Currently, decisions are
+placed individually due to a vote within the blockchain. The discussed
+project is then added or deleted from the dynamic whitelist.
+
+As far as hardware requirements, projects should be able to run on both
+64 and 32 bit OSes. Some BOINC projects are limited to architecture. We
+definitely need a set of guidelines to follow before adding a project to
+the whitelist. It is favorable to choose projects that are accessible
+to the masses.
+
+## How do I delete a project from my list of CPIDs?
+
+We used to support detaching but a vote was made to calculate magnitude
+on every project you still have valid RAC on; so GRC does not support
+detaching projects (it was supposed to prevent gaming the system). There
+is no difference if you score 0 on a projects, as all are taken into
+account equally.
+
+If you insist on not being paid for a specific project, change your team
+away from team Gridcoin for that project.
+
+# Status
+
+## How can I see my Researching Status?
+
+If you have some BOINC credits, you are in team Gridcoin and your email

```

Comment:
`you are in team Gridcoin` same deal about the upcoming team requirement removal